src/chaospy/dist/baseclass.py

```diff
@@ -396,7 +396,7 @@ def __sub__(self, X):
 
     def __rsub__(self, X):
         """Y.__rsub__(X) <==> Y-X"""
-        return chaospy.dist.operators.add(X, -self)
+        return chaospy.dist.operators.add(self, -X)
```

Comment:
I am not familiar with operator overloading or how to do it correctly, but without these fixes only left subtraction/divition/etc works. This fixes at least the newly added tests in ```test_dist.py```